exec/RenderHtml.hs

```diff
@@ -91,6 +90,7 @@ pagesForm f =
     , f (Proxy :: Proxy PageUserSettings)
     , f (Proxy :: Proxy PageAdminSettingsEventsProtocol)
     , f (Proxy :: Proxy PageDelegateVote)
+--    , f (Proxy :: Proxy PageDelegationNetwork) -- FIXME
```

Comment:
no need to take it back in, btw.  i've already done this in my PR.  just curious.